index.html

```diff
@@ -1,3 +1,3 @@
 Hey Hipsters!
 Hey Hackers!
-Hey Hustlers!
\ No newline at end of file
+Hey Hustlers!!!
```

Comment:
Are you angry? Please use only 1 exclamation point.

Reply:
I'm not angry!!! You're angry!!!